process/configuration/regions/example_ES_Las_Palmas_2023.yml

```diff
@@ -495,3 +495,9 @@ reporting:
     "Spanish - Spain":
       'author_names': 'Agregue nombres de autores editando los ajustes de la configuración de la región usando un editor de texto.'
 #########
+## Optional Google Earth Engine API
+## Set 'gee' to true and enter your Google Cloud project ID to initate Google Earth Engine API connection and generate the following indicators in your analysis:
+## 1. Large Public Urban Green Space (LPUGS) accessibility and availability
+## 2. Global Urban Heat Vulnerability Index (GUHVI)
+gee: true
+gee_project_id: ee-global-indicators
```

Comment:
Is this a constant that would never change for users?  If so, we can specify it somewhere else e.g. in the [config.yml ](https://github.com/healthysustainablecities/global-indicators/blob/main/process/configuration/templates/config.yml) template